### PR TITLE
Fixes Z-Wave Mock server add-on

### PR DESCRIPTION
**Pull request by marcelveldt**
- Fix node module resolving not working for mock files
- Fix zwave server not starting due a typo (&& vs &)
- Include some default mock files

@AlCalzone FYI

### CODE REVIEW

File: zwave_mock_server/run.sh [hunk 1]
```diff
@@ -24,8 +39,10 @@ else
...
 fi
 
+
 # start mock driver + zwave-server
+export NODE_PATH=$(npm root --quiet -g)
 bashio::log.info ""
 bashio::log.info "Starting Z-Wave JS Mock Driver + Z-Wave Server..."
-node /usr/local/bin/mock-server -c /config && \
+node /usr/local/bin/mock-server -c /config & \
 node /usr/local/bin/zwave-server tcp://127.0.0.1:5555
```
Comment (agners) [inline zwave_mock_server/run.sh:48]: Hm, if those need to run both, they should actually be two s6 services (`[zwave_mock_server` and `zwave_server` I guess, with the latter having a dependency on the former.

File: zwave_mock_server/config.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: Z-Wave JS Mock Server
-version: 1.0.0
+version: 1.0.1
```
Comment (agners) [inline zwave_mock_server/config.yaml:2]: I'd go for a minor bump at least here.

Comment (AlCalzone) [reply zwave_mock_server/config.yaml:2]: For a bug fix?

File: zwave_mock_server/run.sh (hunk 1 above)
Comment (marcelveldt) [reply zwave_mock_server/run.sh:48]: yeah but this addon is now super simple,  maybe we can make it a bit more structured in follow up prs ?

File: zwave_mock_server/config.yaml (hunk 2 above)
Comment (marcelveldt) [reply zwave_mock_server/config.yaml]: bumped it to 1.1.0

File: zwave_mock_server/run.sh (hunk 1 above)
Comment (agners) [reply zwave_mock_server/run.sh:50]: Right, it is just a `run.sh` right now. Fine by me.

**Review by agners — APPROVED** [review]

File: zwave_mock_server/README.md [hunk 3]
```diff
@@ -3,15 +3,24 @@
...
+For convenience, this add-on also hosts the Z-Wave JS Server directly, making it a all-in-one package to quickly fire up a mocked Z-Wave network. Simply connect to the Z-Wave JS websocket server on port 3000 from the Home Assistant Z-Wave integration.
+
+### Connect Home Assistant to the Mock Server
+
+Within Home Assistant, add a new Integration., click Z-Wave.
+Do not check the 'use default supervisor add-on' checkbox, which allows you to manually enter the URL.
+
+Use this url: `http://77f1785d_zwave_mock_server:3000`
+
+NOTE: If you want to connect from your dveelopment machine to the mock server running on another machine/instance, you need to open the port in the add-on settings for the mock server addon and then you can connect to port 3000 of the host running the mock addon.
```
[the remark this answers is not in the document]
Comment (MartinHjelmare) [reply zwave_mock_server/README.md:15]: addon -> add-on

**Pull request merged**